com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassRaytracingVisibility.hlsl

```diff
@@ -50,7 +51,8 @@ void AnyHitVisibility(inout RayIntersection rayIntersection : SV_RayPayload, Att
     GetSurfaceAndBuiltinData(fragInput, viewWS, posInput, surfaceData, builtinData, currentVertex, rayIntersection.cone, isVisible);
 #if defined(TRANSPARENT_COLOR_SHADOW) && defined(_SURFACE_TYPE_TRANSPARENT)
     // Compute the velocity of the itnersection
-    float3 previousPositionWS = TransformPreviousObjectToWorld(currentVertex.positionOS);
+    float3 positionOS = mul(WorldToObject3x4(), float4(fragInput.positionRWS, 1.0)).xyz - _WorldSpaceCameraPos;
```

Comment:
note: guess it will conflict with the camera relative PR, let's be carful

Reply:
Indeed